isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -636,49 +672,76 @@ namespace Isis {
    *   @history Ian Humphrey - Settings are now read on a project name basis. References #4358.
    *   @history Tyler Wilson 2017-11-02 - Settings now read recent projects.  References #4492.
    *   @history Tyler Wilson 2017-11-13 - Commented out a resize call near the end because it
-   *                                      was messing with the positions of widgets after a
-   *                                      project was loaded.  Fixes #5075.
+   *                was messing with the positions of widgets after a project was loaded.  
+   *                Fixes #5075.
+   *   @history Makayla Shepherd 2018-06-10 - Settings are read from the project root ipce.config.
```

Comment:
This isn't actually a comment on this line. It looks like the comments above here, lines 664 - 667 may need to be updated to describe the new config file location as well.

Reply:
You are correct

Reply:
You are correct. I fixed the issue

Reply:
:+1: Thanks!